protocols/medco/deterministic_switching_test.go

```diff
@@ -1,17 +1,20 @@
 package medco_test
 
 import (
+	"testing"
+	"time"
+
 	"github.com/dedis/cothority/log"
 	"github.com/dedis/cothority/network"
 	"github.com/dedis/cothority/protocols/medco"
 	"github.com/dedis/cothority/sda"
 	. "github.com/dedis/cothority/services/medco/libmedco"
 	"github.com/dedis/crypto/random"
-	"testing"
-	"time"
 )
 
 func TestDeterministicSwitching(t *testing.T) {
+	// see https://github.com/dedis/cothority/issues/479
+	t.Skip()
```

Comment:
General remark: a short message saying why this is skipped (and ideally that it will be resolved by XXX) would be nice